docs/content/manual/manual.md

```diff
@@ -0,0 +1,5538 @@
+like addition but if both objects contain a value for the
+same key, and the values are objects, the two are merged with
+the same strategy.
+
+::: Examples
+
+~~~
+10 / . * 3
+5
+6
+~~~
+
+~~~
+. / ", "
+"a, b,c,d, e"
+["a","b,c,d","e"]
+~~~
+
+~~~
+{"k": {"a": 1, "b": 2}} * {"k": {"a": 0,"c": 3}}
+null
+{"k": {"a": 0, "b": 2, "c": 3}}
+~~~
+
+~~~
+.[] | (1 / .)?
+[1,0,-1]
+1
+-1
+~~~
+
+:::
+
+## Equality: `==`, `!=`
+
+The expression `a == b` produces
+`true` if the results of evaluating `a` and `b` are equal
+(that is, if they represent equivalent JSON values) and
+`false` otherwise.
+In particular, strings are never considered equal to numbers.
+In checking for the equality of JSON objects, the ordering of keys is irrelevant.
+If you're coming from JavaScript, please note that jq's `==` is like
+JavaScript's `===`, the "strict equality" operator.
+
+The expression `a != b` returns `false` if `a == b` returns `true`, else `true`.
+
+::: Examples
+
+~~~
+. == false
+null
+false
+~~~
+
+~~~
+. == {"b": {"d": (4 + 1e-20), "c": 3}, "a":1}
+{"a":1, "b": {"c": 3, "d": 4}}
+true
+~~~
+
+~~~
+.[] == 1
+[1, 1.0, "1", "banana"]
+true
+true
+false
+false
+~~~
+
+:::
+
+## Ordering: `>`, `>=`, `<=`, `<` {#ordering}
+
+The ordering operators `>`, `>=`, `<=`, `<` return whether
+their left argument is greater than, greater than or equal
+to, less than or equal to or less than their right argument
+(respectively).
+
+If two values of different type are ordered, the
+types are ordered instead in the following increasing order:
+
+* `null`
+* booleans
+* numbers
+* strings
+* arrays
+* objects
+
+For booleans, `false` is smaller than `true`.
+Strings are ordered [alphabetically](https://en.wikipedia.org/wiki/Alphabetical_order)
+(by Unicode codepoint value).
+Arrays are ordered [lexicographically](https://en.wikipedia.org/wiki/Lexicographic_order).
+The ordering for objects is a little complex: first they're
+compared by comparing their sets of keys (as arrays in
+sorted order), and if their keys are equal then the values
+are compared key by key.
+
+::: Examples
+
+~~~
+. < 5
+2
+true
+~~~
+
+~~~
+[1,3] > [1,2]
+null
+true
+~~~
+
+~~~
+"jq" > false
+null
+true
+~~~
+
+:::
+
+
+# Boolean filters
+
+Every value can be converted to a boolean --- in particular,
+the values `false` and `null` have the _boolean value_ `false`,
+all other values have the boolean value `true`.
+This section describes several filters that analyze the boolean value of values.
+
+You can negate the boolean value of a value with the builtin function [`not`](#not).
+It is called as a filter to which things can be piped
+rather than with special syntax, as in `.foo and .bar | not`.
+
+::: Note
+
+Mind the precedences.
+In particular, the filter
+`false and false | not` is equivalent to
+`(false and false) | not` and yields `true`, whereas
+`false and (false | not)` yields `false`.
+
+:::
+
+## if-then-else-end {#if-then-else}
```

Comment:
Anchor is `if-then-else-end`. I think `end` keyword is necessary.

Reply:
Thanks for spotting this!